src/lightning_app/core/app.py

```diff
@@ -390,6 +392,7 @@ def run_once(self):
         self._update_layout()
         self.maybe_apply_changes()
 
+        # TODO: This should be removed
```

Comment:
Let's have the TODO mention when to remove it. I think you meant to say that it should be removed once the new logic is no longer experimental.